ivy/functional/backends/paddle/elementwise.py

```diff
@@ -444,48 +309,31 @@ def multiply(
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
     x1, x2, ret_dtype = _elementwise_helper(x1, x2)
-    if x1.dtype in [paddle.int8, paddle.int16, paddle.uint8, paddle.float16]:
-        x1, x2 = x1.astype("float32"), x2.astype("float32")
     return paddle.multiply(x1, x2).astype(ret_dtype)
 
 
-@with_unsupported_device_and_dtypes(
-    {"2.5.1 and below": {"cpu": ("bool", "bfloat16")}},
-    backend_version,
-)
+@with_supported_dtypes({"2.5.1 and below": "float"}, backend_version)
```

Comment:
as `bfloat16` was previously marked unsupported in the function, could you please check if it's supported now (as we've marked `float` as supported?

Reply:
no it's not